services/src/main/java/org/keycloak/services/resources/admin/AdminRoot.java

```diff
@@ -277,4 +302,11 @@ private static Properties getMessages(KeycloakSession session, RealmModel realm,
         }
     }
 
+    private static boolean isAdminApiEnabled() {
```

Comment:
These methods could be moved somewhere else to be shared as public static methods.